modules/tensor_mechanics/src/actions/DomainIntegralAction.C

```diff
@@ -84,6 +84,7 @@ validParams<DomainIntegralAction>()
 DomainIntegralAction::DomainIntegralAction(const InputParameters & params)
   : Action(params),
     _boundary_names(getParam<std::vector<BoundaryName>>("boundary")),
+    _use_crack_front_points_provider(false),
```

Comment:
You added a new unsigned int but didn't initialize it in all branches in the constructor. That may not bite you today, but a little refactor and you could be accessing uninitialized values.

Reply:
are you talking about _use_crack_front_points_provider? I think I have initialized it in the Constructor.

Reply:
No sorry, I'm talking about the "num crack front points" variable. This is just where that initialization should go. Alternatively, you can initialize it to zero (or some other sensible value) in all branches in your constructor body.